diff --git a/node_modules/@electron/get/dist/cjs/artifact-utils.js b/node_modules/@electron/get/dist/cjs/artifact-utils.js
index 5766965..7f062c2 100644
--- a/node_modules/@electron/get/dist/cjs/artifact-utils.js
+++ b/node_modules/@electron/get/dist/cjs/artifact-utils.js
@@ -1,8 +1,8 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 const utils_1 = require("./utils");
-const BASE_URL = 'https://github.com/electron/electron/releases/download/';
-const NIGHTLY_BASE_URL = 'https://github.com/electron/nightlies/releases/download/';
+const BASE_URL = 'https://github.com/HyperPlay-Gaming/electron-releases/releases/download/';
+const NIGHTLY_BASE_URL = 'https://github.com/HyperPlay-Gaming/electron-releases/releases/download/';
 function getArtifactFileName(details) {
     utils_1.ensureIsTruthyString(details, 'artifactName');
     if (details.isGeneric) {
diff --git a/node_modules/@electron/get/dist/esm/artifact-utils.js b/node_modules/@electron/get/dist/esm/artifact-utils.js
index 8dc1433..fe03740 100644
--- a/node_modules/@electron/get/dist/esm/artifact-utils.js
+++ b/node_modules/@electron/get/dist/esm/artifact-utils.js
@@ -1,6 +1,6 @@
 import { ensureIsTruthyString, normalizeVersion } from './utils';
-const BASE_URL = 'https://github.com/electron/electron/releases/download/';
-const NIGHTLY_BASE_URL = 'https://github.com/electron/nightlies/releases/download/';
+const BASE_URL = 'https://github.com/HyperPlay-Gaming/electron-releases/releases/download/';
+const NIGHTLY_BASE_URL = 'https://github.com/HyperPlay-Gaming/electron-releases/releases/download/';
 export function getArtifactFileName(details) {
     ensureIsTruthyString(details, 'artifactName');
     if (details.isGeneric) {
