diff --git a/node_modules/@wagmi/core/dist/esm/actions/switchChain.js b/node_modules/@wagmi/core/dist/esm/actions/switchChain.js
index d91f5b1..90177de 100644
--- a/node_modules/@wagmi/core/dist/esm/actions/switchChain.js
+++ b/node_modules/@wagmi/core/dist/esm/actions/switchChain.js
@@ -4,8 +4,12 @@ import { SwitchChainNotSupportedError, } from '../errors/connector.js';
 export async function switchChain(config, parameters) {
     const { addEthereumChainParameter, chainId } = parameters;
     const connection = config.state.connections.get(parameters.connector?.uid ?? config.state.current);
-    if (connection) {
-        const connector = connection.connector;
+    let connector = connection?.connector
+    if (!connector && config.connectors && config.connectors.length){
+        console.warn('Config state connections does not have a valid connector. Setting connector from config connectors directly.')
+        connector = config.connectors[0]
+    }
+    if (connector) {
         if (!connector.switchChain)
             throw new SwitchChainNotSupportedError({ connector });
         const chain = await connector.switchChain({
