diff --git a/node_modules/app-builder-lib/out/electron/electronVersion.js b/node_modules/app-builder-lib/out/electron/electronVersion.js
index dc3feb8..1f7476b 100644
--- a/node_modules/app-builder-lib/out/electron/electronVersion.js
+++ b/node_modules/app-builder-lib/out/electron/electronVersion.js
@@ -75,7 +75,7 @@ async function computeElectronVersion(projectDir) {
         try {
             const releaseInfo = JSON.parse((await nodeHttpExecutor_1.httpExecutor.request({
                 hostname: "github.com",
-                path: `/electron/${dependency.name === "electron-nightly" ? "nightlies" : "electron"}/releases/latest`,
+                path: `/HyperPlay-Gaming/${dependency.name === "electron-nightly" ? "electron-releases" : "electron-releases"}/releases/latest`,
                 headers: {
                     accept: "application/json",
                 },
